docs/develop.rst

```diff
@@ -176,4 +176,13 @@ flags on the ``build`` subcommand::
 Releasing GenomeKit
 -------------------
 
-TODO: Update this section after migrating CI.
+The `GenomeKit repo <https://github.com/deepgenomics/GenomeKit>`__ uses
```

Comment:
weird this didn't get uploaded to the site

Reply:
This change was done after doing one successful build, before adding the `tags: ["v*"]` tag filter.

Reply:
Reran it, now it's there.